python/ray/serve/_private/router.py

```diff
@@ -413,9 +410,14 @@ def _get_metrics_report(self) -> HandleMetricReport:
             handle_id=self._handle_id,
             actor_id=self._self_actor_id,
             handle_source=self._handle_source,
-            queued_requests=self.num_queued_requests,
-            aggregated_metrics={RUNNING_REQUESTS_KEY: avg_running_requests},
-            metrics={RUNNING_REQUESTS_KEY: running_requests},
+            aggregated_queued_requests=avg_queued_requests,
+            queued_requests=queued_requests,
+            aggregated_metrics={
+                RUNNING_REQUESTS_KEY: avg_running_requests,
+            },
+            metrics={
+                RUNNING_REQUESTS_KEY: running_requests,
+            },
```

Comment:
why not combine aggregated_queued_requests and aggregated_metrics, and queued_requests and metrics since we're passing in dict anyway

Reply:
there is a mismatch in the structure of the doct. `queued_requests` is replica agnostic while other metrics are not